docs/docs/events.md

```diff
@@ -133,17 +133,45 @@ commandCompletedListener.remove();
 |**completionTime**|Timestamp when the command, and consecutive animations, completed.|
 
 ## registerModalDismissedListener
-Invoked when modal dismissed.
+
+Called each time this component (Modal) is dismissed. 
```

Comment:
I find this description a bit misleading as a modal can be dismissed only once. This method is invoked each time a modal (any modal) is dismissed. I think we should keep the current description as the new description is misleading

Reply:
Done!